src/main/java/com/fmi/tournament/organizer/service/RegistrationService.java

```diff
@@ -0,0 +1,72 @@
+package com.fmi.tournament.organizer.service;
+
+import com.fmi.tournament.organizer.exception.*;
+import com.fmi.tournament.organizer.model.*;
+import com.fmi.tournament.organizer.repository.ParticipantRepository;
+import com.fmi.tournament.organizer.repository.TournamentRepository;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+@Service
+public class RegistrationService {
+    private final TournamentRepository tournamentRepository;
+    private final ParticipantRepository participantRepository;
+
+    public RegistrationService(TournamentRepository tournamentRepository, ParticipantRepository participantRepository) {
+        this.tournamentRepository = tournamentRepository;
+        this.participantRepository = participantRepository;
+    }
+
+    public void registration(UUID participantID, UUID tournamentID){
```

Comment:
It is a good practice names of methods to be verbs. For example "register" instead of "registration" and "unregister" instead of "unregistration".